javacord-api/src/main/java/org/javacord/api/entity/server/Server.java

```diff
@@ -49,6 +51,70 @@
  */
 public interface Server extends DiscordEntity, Nameable, UpdatableFromCache<Server>, ServerAttachableListenerManager {
 
+
+    /**
+     * Gets the features of the server.
+     *
+     * @return The servers available features.
+     */
+    Collection<ServerFeature> getFeatures();
+
+    /**
+     * Gets the premium tier of the server.
+     *
+     * @return The premium tier.
+     */
+    PremiumTier getPremiumTier();
+
+    /**
+     * Gets the premium subscription count of the server.
+     *
+     * @return The premium subscription count.
+     */
+    int getPremiumSubscriptionCount();
```

Comment:
Is this the amount of nitro boosts?
If yes, this should be
```suggestion
    int getServerBoostCount();
```